src/lib/wasm/ConstellationStore.ts

```diff
@@ -13,8 +13,8 @@ import prettyBytes from "pretty-bytes"
 class ConstellationStore {
     private constellationWritable: Writable<wasm.ConstellationBox | null>
     public freeStorageSpace: Writable<string> = writable("Loading...")
-    public MAX_FILE_SIZE = 104857600
-    public MAX_STORAGE_SIZE = 2 * 1024 * 1024 * 1024
+    public MAX_FILE_SIZE = 2000 * 1024 * 1024 // 20 GB
```

Comment:
I suggest following the same pattern used by MAX_STORAGE_SIZE:
```
2 * 1024 * 1024 * 1024 // 2 GB
```
Also, fixing the comment ;)

Reply:
updated 😄